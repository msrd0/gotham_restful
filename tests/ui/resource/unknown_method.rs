#[macro_use]
extern crate gotham_restful;

#[derive(Resource)]
#[resource(read_any)]
struct FooResource;

#[read_all]
fn read_all() {}

fn main() {}
