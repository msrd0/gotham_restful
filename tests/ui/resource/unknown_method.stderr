error[E0412]: cannot find type `read_any___gotham_restful_endpoint` in this scope
 --> tests/ui/resource/unknown_method.rs:5:12
  |
5 | #[resource(read_any)]
  |            ^^^^^^^^
...
8 | #[read_all]
  | ----------- similarly named struct `read_all___gotham_restful_endpoint` defined here
  |
help: a struct with a similar name exists
  |
5 | #[resource(read_all___gotham_restful_endpoint)]
  |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
help: you might be missing a type parameter
  |
4 | #[derive(Resource<read_any___gotham_restful_endpoint>)]
  |                  ++++++++++++++++++++++++++++++++++++
